Extensions/classic_tags.js

```diff
@@ -82,6 +83,97 @@ XKit.extensions.classic_tags = new Object({
 		$(this).attr("href", newHref);
 	},
 
+	get_post_timestamp: function(blog_name, post_id) {
+		var self = this;
+		var api_url = "https://api.tumblr.com/v2/blog/" + blog_name + "/posts" + "?api_key=" + self.apiKey + "&id=" + post_id;
+		var promise = $.Deferred();
+
+		function fail() {
+			XKit.console.add('Unable to fetch timestamp for post ' + post_id);
```

Comment:
Opted for console.log, users aren't going to be able to do anything about these errors